train_gpt2.cu

```diff
@@ -1212,7 +1219,7 @@ void gelu_forward(floatX* out, const floatX* inp, int N) {
 
 void gelu_backward(floatX* dinp, const floatX* inp, const floatX* dout, const int N) {
     const int block_size = 128;
-    const int grid_size = CEIL_DIV(N, block_size);
+    const int grid_size = CEIL_DIV(N, (int)(block_size * x128::size));
```

Comment:
(int) shouldn't be necessary anymore, IIRC we changed  x128::size  to be of type int.